languages/csharp/reference/exercise-concepts/high-scores.md

```diff
@@ -0,0 +1,29 @@
+# Concepts of high-scores
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/high-scores/Example.cs)
+
+## General concepts
+### Object-oriented
+- classes: 
+    - the tested methods are defined in a class
+- methods: 
+    - methods arguments: the time is passed as an argument
+    - return values: returning a value from a method
+
+### General
+- scoping
+    - use `{` and `}` to denote scoping
+    - visibility: making tested method and tested class `public`
+    - imports: import types through `using` statements (collections, LINQ)
+
+### Types
+- numbers: 
+    - signed integers: used for score values
+    - lists: strongly typed list of `int` objects 
+
+### Functional
```

Comment:
I might be nit picking here, but this exercise is solvable without using LINQ (although the LINQ solution is shorter). You can simplify iterate to find the max, sort to find the top three, etc.

Given that, I wonder if it makes sense to move this to describe one approach and loops as another one.

Reply:
Sure, it is solvable in both ways - I will include also the non-LINQ approach in the concepts list.

Reply:
Thanks, fixed